templates/lab-results/templates/main.tpl

```diff
@@ -0,0 +1,144 @@
+{{define "test-result-header"}}
+    <table-cell background-color="#1B7A98" align="center" vertical-align="middle">
+        <paragraph text-align="center">
+            <text-chunk font-size="10" color="#ffffff">{{ .Text }}</text-chunk>
+        </paragraph>
+    </table-cell>
+{{end}}
+
+{{define "test-result-row"}}
+    <table-cell align="center" vertical-align="top" border-width-bottom="1" border-color="#1B7A98">
```

Comment:
No need to specify the `vertical-align` attribute as `top` is the default value for that.